src/app/notifications/page.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { User } from "lucide-react";
+import Navbar from "../_components/Navbar";
+
+interface Notification {
+	id: number;
+	message: string;
+	action: string;
+}
+
+const mockNotifications: Notification[] = [
```

Comment:
Using a mock array is a nice approach for development and testing, as we want to see the notifications when running the app and we have no connection with the backend. However, given that the goal is to connect with the backend asap, I would suggest commenting it out or removing it, as we are not using this in production.